ci-operator/jobs/openshift/installer/openshift-installer-release-4.5-presubmits.yaml

```diff
@@ -588,7 +591,6 @@ presubmits:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-installer-release-4.5-e2e-gcp-upgrade
-    optional: true
```

Comment:
Can we confirm that this one is really desired to be *not* optional?

Reply:
Checked on master: It is NOT optional

https://github.com/openshift/release/blob/5d8e151961b0aa3b1c6cf245c581ab0597392d42/ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml#L1090-L1153

Reply:
What I mean is: apparently somebody manually set this on the release branch, I don't think we should overwrite it.

Reply:
I understand it is not optional on master, but that does not mean it should also be not-optional on release branch.

Reply:
Got it.
Keep it unchanged.